avvoenv.1.ronn

```diff
@@ -49,6 +53,14 @@ current service, or canonicalise the name given with the `--service` option.
     and/or with a list of multiple space-separated key value pairs. In the
     second case the argument must be quoted.
 
+  * `--app-id` [<app-id>]:
+    Authenticate with Vault via app-id. If the <app-id> argument is provided
+    it will override the `VAULT_APP_ID` environment variable.
+
+  * `--app-user` [<app-user>]:
```

Comment:
Sorry I missed it first time round, can these include the short option too.

I'm not sure if it makes most sense to order by the short or long option. Whichever makes sense to you.

Reply:
👍  Ive added the short options and am now ordering by short option.